server/konnectors/weboob.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+/**
+ * A Konnector to communicate with a Weboob instance.
+ *
+ * You need a running [cozyweboob](https://github.com/Phyks/cozyweboob)
+ * somewhere.
+ *
+ * // TODO: What happens when moment is an invalid date?
+ */
+
+// NPM imports
+const moment = require('moment');
+const requestJson = require('request-json');
+
+// Konnectors imports
+const baseKonnector = require('../lib/base_konnector');
+const saveDataAndFile = require('../lib/save_data_and_file');
+
+// Models imports
+// TODO: Add support for others models
+const Bill = require('../models/bill');
+
+
+/**
+ * Converters between exported data and available models in Cozy Konnectors.
+ *
+ * Maps all the available weboob types in the exported JSON to function
+ * exporting a matching Cozy Model.
+ */
+const Converter = {
+    // Conversion functions for CapDocument items to Bill
+    'subscriptions': function (data, moduleName) {  // Weboob type: Subscription
+        // Do nothing for subscriptions
+        return {
+            cozyModel: undefined,
+            parsedData: undefined,
+        };
+    },
+    'bills': function (data, moduleName) {  // Weboob type: Bill
+        var parsedBills = [];
+        Object.keys(data).forEach(function (subscriptionID) {
+            data[subscriptionID].forEach(function (bill) {
+                // TODO: Label not mapped
+                parsedBills.push({
+                    type: '',  // TODO: What is it?
+                    subtype: '',  // TODO: What is it?
+                    date: moment(bill.date),
+                    vendor: moduleName,
+                    amount: parseFloat(bill.price),
+                    vat: parseFloat(bill.vat),
+                    currency: bill.currency,
+                    plan: '',  // TODO: What is it?
+                    pdfurl: bill.url,
```

Comment:
There is a bug there at the moment, as we always store the URL, even if we [only asked for a fetch](https://github.com/Phyks/cozyweboob#input-json-file). In the `fetch` only case, the URL is set to the original URL on the vendor site, and we basically end up downloading a login page.